<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thông tin khách hàng</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow {
        background-color: #fef8ec;
    }
    
    QWidget#centralwidget {
        background-color: #fef8ec;
        border-radius: 8px;
        margin: 20px;
        padding: 20px;
    }
    
    QLabel#label_overview {
        font-family: Arial;
        font-size: 24px;
        font-weight: bold;
        color: #333333;
        padding: 10px 0;
        border-bottom: 2px solid #e0e0e0;
        margin-bottom: 20px;
    }
    
    QLabel#label_photo {
        background-color: #f8f9fa;
        border: 2px solid #e0e0e0;
        border-radius: 75px;
        padding: 5px;
    }
    
    QLineEdit {
        padding: 8px;
        border: 1px solid #e0e0e0;
        border-radius: 4px;
        background-color: #ffffff;
        font-size: 14px;
        min-height: 25px;
    }
    
    QLineEdit:focus {
        border: 2px solid #4a90e2;
        outline: none;
    }
    
    QLineEdit:hover {
        border: 1px solid #4a90e2;
    }
    
    QTextEdit {
        padding: 8px;
        border: 1px solid #e0e0e0;
        border-radius: 4px;
        background-color: #ffffff;
        font-size: 14px;
    }
    
    QTextEdit:focus {
        border: 2px solid #4a90e2;
        outline: none;
    }
    
    QDateEdit {
        padding: 8px;
        border: 1px solid #e0e0e0;
        border-radius: 4px;
        background-color: #ffffff;
        min-height: 25px;
    }
    
    QLabel {
        color: #333333;
        font-size: 14px;
        font-weight: 500;
    }
    
    QPushButton {
        background-color: #4a90e2;
        color: white;
        border: none;
        border-radius: 4px;
        padding: 8px 16px;
        font-size: 14px;
    }
    
    QPushButton:hover {
        background-color: #357abd;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_overview">
      <property name="text">
       <string>Thông tin khách hàng</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_photo">
      <item>
       <widget class="QLabel" name="label_photo">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string>Ảnh</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_choosePhoto">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Chọn ảnh</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_hoTen">
        <property name="text">
         <string>Họ tên: *</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_hoTen"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_sdt">
        <property name="text">
         <string>Số điện thoại: *</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_sdt"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_email">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_email"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_diaChi">
        <property name="text">
         <string>Địa chỉ:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTextEdit" name="textEdit_diaChi"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_ngayDangKy">
        <property name="text">
         <string>Ngày đăng ký:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDateEdit" name="dateEdit_ngayDangKy">
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_buttons">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_save">
        <property name="text">
         <string>Lưu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="text">
         <string>Hủy</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
